common/logger/src/struct_log.rs

```diff
@@ -86,7 +86,7 @@ pub struct StructuredLogEntry {
 
 impl StructuredLogEntry {
     /// Base implementation for creating a log
-    fn new_unnamed() -> Self {
+    pub fn new_unnamed() -> Self {
```

Comment:
intentional, this doesn't seem used in this commit.

Reply:
Is there a reason we need the non-categorized version?

Reply:
If you noticed, this is so that the schema proc-macro can easily create one of these Types and the category is just set by the macro. I'm very open to improvements on this front though

Reply:
Yeah, this whole thing can be worked around later.

I was planning on removing this "unnamed" vs "named" and just make one.